scripts/executors/local-codepen.js

```diff
@@ -23,8 +23,9 @@ if (fs.existsSync(configPath)) {
   try {
     console.log("Attempting to npm link globally installed ngrok so it can be require'd");
     // This will probably install ngrok globally if it's not already present
-    execSync('npm link ngrok@3', { cwd: gitRoot, stdio: 'inherit' });
+    execSync('npm link ngrok@5 --force', { cwd: gitRoot, stdio: 'inherit' });
```

Comment:
Thanks for the POC.
Lets move the changes to local-codepen.js file and .gitignore file to a separate PR and push the changes only in react-charting package.
I will take a look how to make use of the existing script and if there are any prerequisites that we need to setup to be able to use the common script with minimal changes.

Reply:
Ok....
Moving script to charting package then

Reply:
dont move local-codepen.js script to charting package. Just move it to a different PR.

Reply:
Done